packages/block-editor/src/components/list-view/use-list-view-images.js

```diff
@@ -12,19 +11,40 @@ import { store as blockEditorStore } from '../../store';
 
 // Maximum number of images to display in a list view row.
 const MAX_IMAGES = 3;
+const IMAGE_GETTERS = {
```

Comment:
Not a blocker, but I need to come up with a way to add Gallery getter here as well, to showcase that the new public API getter can work with all sorts of blocks.